components/src/dynamo/frontend/tests/test_vllm_processor_unit.py

```
@@ -126,6 +126,165 @@ def test_no_tools_in_request(self, tokenizer):
         ), "No tools in request should produce None tools in template"
 
 
+class TestMultimodalFeatureMetadata:
+    def _feature(
+        self, modality, mm_hash, offset, length, data=object(), is_embed=None
```

Ruff (0.15.15)

[warning] Do not perform function call `object` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)